src/app/components/superhero/Superhero.tsx

```diff
@@ -0,0 +1,32 @@
+import { Error } from '@app/components/shared/error/Error';
+import { PageHeader } from '@app/components/shared/pageHeader/PageHeader';
+import { Spinner } from '@app/components/shared/spinner/Spinner';
+import { SuperheroDetails } from '@app/components/superhero/components/SuperheroDetails';
+import { useSuperhero } from '@app/components/superhero/useSuperhero';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+
+const Superhero = () => {
+  const { t } = useTranslation();
+  const { id } = useParams<{ id: string }>();
+  const { isLoading, isError, superhero } = useSuperhero(parseInt(id ?? '0'));
```

Comment:
**question** when id will be empty string or string that doesn't contain digits NaN will be passed to hook.  Is this intentional?